_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1426,58 +1459,149 @@ Set the variables for the `account-baseline-app` module in this environment in t
 .infrastructure-live/logs/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
+locals {
+  accounts = {
+    logs     = "409800740445"
+    security = "673123100581"
+    shared   = "384759303421"
+    dev      = "293847503945"
+    stage    = "384924092834"
+    prod     = "784260063686"
+    root     = "216044045972"
+  }
+
+  # Both buckets are created in the logs account by account-baseline-root
+  config_s3_bucket_name     = "acme-config-bucket-logs"
+  cloudtrail_s3_bucket_name = "acme-cloudtrail-logs"
+
+  # The Cloudtrail KMS Key is deployed at the logs account but it's value is an output from the root account.
+  cloudtrail_kms_key_arn = "arn:aws:kms:us-east-1:409800740445:alias/cloudtrail-acme"
+
+  # A local for convenient access to the security account root ARN.
+  security_account_root_arn = "arn:aws:iam::${local.accounts.security}:root"
+}
+
 inputs = {
-  # Fill in your region you want to use (only used for API calls) and the ID of your logs AWS account
-  aws_region     = "us-east-2"
-  aws_account_id = "<LOGS_ACCOUNT_ID>"
+  # Fill in your region you want to use (only used for API calls) and the ID of your root AWS account
+  aws_region     = "us-east-1"
+  aws_account_id = local.accounts["logs"]
 
   # Prefix all resources with this name
-  name_prefix    = "<COMPANY_NAME>-logs"
+  name_prefix = "acme-logs"
+ ################################
+  # Parameters for AWS Config
+  ################################
+  # Send Config logs to the common S3 bucket.
+  config_s3_bucket_name = local.config_s3_bucket_name
 
-  # Set to true to create an sns topic in this account for sending AWS Config notifications (e.g., if this is the logs account).
-  config_should_create_sns_topic = true
+  # Send Config logs and events to the logs account.
+  config_central_account_id = local.accounts.logs
 
-  # Use the S3 bucket that was already created in this logs account by account-baseline-root
-  config_s3_bucket_name = "<CONFIG_BUCKET_NAME>"
+  # Do not allow objects in the Config S3 bucket to be forcefully removed during destroy operations.
+  config_force_destroy = false
 
-  # Set this to false because this is the account to aggregate Config data
-  config_aggregate_config_data_in_external_account = false
-  config_central_account_id                        = null
+  #  This is the Logs account, so we create the SNS topic for aggregating Config logs from all accounts.
+  config_should_create_sns_topic = true
 
-  # The bucket was created by account-baseline-root in this account
-  config_should_create_s3_bucket = false
+  # All of the other accounts send logs to this account.
+  config_linked_accounts = [
+  for name, id in local.accounts :
+  id if name != "logs"
```

Comment:
NIT: In my view, this is a line under the for loop, so it seems like it should be further indented. (We need to update the ref arch generation to reflect this too.)